src/Queue.js

```diff
@@ -1,23 +1,32 @@
 // @flow
-import BaseModel from './models/BaseModel';
+import type { LandRequest } from './types';
 
 export default class Queue {
-  client: Object;
+  queue: Array<LandRequest>;
 
-  constructor(client: Object) {
-    this.client = client;
+  constructor() {
+    this.queue = [];
   }
 
-  async list(name: string) {
-    let items = await this.client.lrangeAsync(name, 0, -1);
-    return items.map(item => JSON.parse(item));
+  list() {
```

Comment:
Can you actually leave the methods in here async? In the future it makes sense that we'd want to do something synchronous and its just easier if we always act like it is

Reply:
Will do.